k2/csrc/ragged_ops_inl.h

```diff
@@ -540,11 +540,13 @@ void ArgMaxPerSublist(Ragged<T> &src, T initial_value, Array1<int32_t> *dst) {
 
   int32_t last_axis = src.NumAxes() - 1;
   const Array1<int32_t> &row_splits_array = src.RowSplits(last_axis);
+  const Array1<int32_t> &row_ids_array = src.RowIds(last_axis);
```

Comment:
Stuff related to row_ids can be moved to the block belonging to the `#if 0` macro.